runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -8250,6 +8255,28 @@ done:;
 	}
 
 #if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+	VM_BytecodeAction
+	defaultValueWithUnflattenedFlattenables(J9Class *clazz, j9object_t instance)
+	{
+		J9FlattenedClassCacheEntry * entry;
+		UDATA length = clazz->flattenedClassCache->numberOfEntries;
+		for (UDATA index = 0; index < length; index++) {
+			entry = J9_VM_FCC_ENTRY_FROM_CLASS(clazz, index);
+			if (!J9_ARE_ALL_BITS_SET(J9ClassIsFlattened, entry->clazz->classFlags)) {
+				if (entry->offset == 0) {
+					J9Class *definingClass;
+					J9ROMFieldShape *field;
+					J9ROMNameAndSignature * nameAndSig = entry->nameAndSignature;
+					J9UTF8 *name = J9ROMNAMEANDSIGNATURE_NAME(nameAndSig);
+					J9UTF8 *signature = J9ROMNAMEANDSIGNATURE_SIGNATURE(nameAndSig);
+					entry->offset = instanceFieldOffset(_currentThread, clazz, J9UTF8_DATA(name), J9UTF8_LENGTH(name), J9UTF8_DATA(signature), J9UTF8_LENGTH(signature), &definingClass, (UDATA *)&field, 0);
+				}
+				J9STATIC_OBJECT_STORE(_currentThread, clazz, entry->offset, entry->clazz->flattenedClassCache->defaultValue);
```

Comment:
I dont think the tests actually work since you havent stored the defaultValue in the `instance`. You should be using to store the defaultValue.

Once you've added a test for Line, paste a memory dump of the contents on this PR